### PR TITLE
(chore) revert to community patient-labs and update related concepts

### PR DESCRIPTION
**Pull request by donaldkibet**
### Description

Revert back to community version for patient-labs-app and updated related lab orderable concept uuid 

See https://github.com/openmrs/openmrs-esm-patient-chart/pull/1794

### CODE REVIEW

File: configuration/dev-config.json [hunk 1]
```diff
@@ -612,6 +612,7 @@
       }
     ],
     "orders": {
+      "labOrderableConcepts": ["1271AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"],
```
Comment (ojwanganto) [inline configuration/dev-config.json:615]: how is this used in the module?

**Review by ojwanganto — APPROVED** [review]
Looks good.

File: configuration/dev-config.json (hunk 1 above)
Comment (donaldkibet) [reply configuration/dev-config.json:615]: This is used to fetch all the test we want to order. 
Screenshot from 2024-04-19 09-04-49

Comment (donaldkibet) [reply configuration/dev-config.json:615]: The community module expects setMembers while this concept has answers with the relevant tests. The PR i have done  on the community instance adds the ability to also include `answers` for the test orders

**Pull request merged**